source/Octopus.Cli/Commands/Runbooks/RunRunbookCommand.cs

```diff
@@ -197,13 +217,14 @@ protected override Task ValidateParameters()
             }
 
             if ((RunAt ?? DateTimeOffset.Now) > NoRunAfter)
-                throw new CommandException("The Run will expire before it has a chance to execute.  Please select an expiry time that occurs after the deployment is scheduled to begin");
+                throw new CommandException(
+                    "The Run will expire before it has a chance to execute.  Please select an expiry time that occurs after the deployment is scheduled to begin");
 
             CheckForIntersection(IncludedMachineIds.ToList(), ExcludedMachineIds.ToList());
 
-            if (TenantIds.Contains("*") && (TenantIds.Count > 1 || TenantTagNames.Count > 0))
+            if (TenantNamesOrIds.Contains("*") && (TenantNamesOrIds.Count > 1 || TenantTagNames.Count > 0))
                 throw new CommandException(
-                    "When running on all tenants using the --tenantIds=* wildcard, no other tenant filters can be provided");
+                    "When running on all tenants using the --tenant=* wildcard, no other tenant filters can be provided");
```

Comment:
Is there a test we need to update in line with the param name change, or has that already happened?

Reply:
I think the param change was already done, but the exception message text hadn't been updated.